FastSimulation/Tracking/python/SeedingMigration.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+
+def _regionProducerToFactoryPSet(producer):
```

Comment:
I'd rather this file and its functionality does not exist.
This is effectively an in-situ "patch < a.diff" written in a more fancy way.